### ISSUE TITLE
Markdown syntax highlighting?

### ISSUE THREAD

**Issue by bramtayl** [julia-vscode/julia-vscode#1641]
Is there a way to outsource docstrings to VSCode's markdown editor? Would go nicely with #1426. Apologies if there's already an issue open for this.

**Comment by davidanthoff** [comment]
What do you mean by "outsource docstrings to VSCode's markdown editor"? Syntax highlighting? Something else?

**Comment by bramtayl** [comment]
Yup, syntax highlighting and other formatting. VSCode seems to come with some built in support for markdown; you can see some examples here: https://code.visualstudio.com/docs/languages/markdown

**Comment by pfitzseb** [comment]
Pretty sure we had that at some point (at least in the Atom grammar). Turned out to be a bad idea because the markdown highlighting would regularly leak into Julia code. It's possible that VSCode's implementation doesn't do that though.

**Comment by bramtayl** [comment]
Why was it leaking into Julia code? It would be unavoidable if there were mismatched quotes, but otherwise that might have been a bug somewhere?
